playground/app/components/todo-item.component.ts

```diff
@@ -12,9 +12,19 @@ export class TodoItemCmp implements OnChanges{
 
   @Input('<') todo: TodoModel;
   @Input('<') idx: number;
-  @Output() onDone: ( todo: {todo:TodoModel} )=>void;
+  // this is old way how to bindings are created, will be removed in 2.0
+  // @Output() onDone: ( todo: {todo:TodoModel} )=>void;
+  @Output() onDone: EventEmitter<TodoModel>;
+  @Output() emitTest = new EventEmitter<string>();
 
-  ngOnInit(){}
+  ngOnInit(){
+
+    const dispose = this.onDone.subscribe( (value)=> {
+      console.log( `onDone emitted value: ${value}` );
```

Comment:
This code is only to demostrate that you implemented the subscribe method on the EventEmitter conforming to RxJS, but I still don't understand why you want it on the "client side" why a component has to register and subscribe to the event that emit itself?
In NG2 I see the value of this approach to handle events but on the *parent code* (comp that handle the event using subscribe + RxJS operators) but not here... 
If I correctly understood your code, even if you've created the EventEmitter class I can use it only on this side!? Or I'm missing something / don't undestood it?